ui/components/ui/new-network-info/index.scss

```diff
@@ -0,0 +1,52 @@
+.new-network-info {
+  &__wrapper {
+    box-shadow: 0 4px 20px rgba(0, 0, 0, 0.214);
+    border-radius: 8px;
+
+    .popover-footer {
+      border-top: none;
+    }
+
+    .popover-header {
+      padding-bottom: 1px;
+    }
+
+    .fa-question-circle,
+    .popover-header__button {
+      color: var(--ui-4);
+    }
+  }
+
+  &__token-box {
+    align-self: center;
+    margin-top: 8px;
+    max-width: 245px;
+
+    .typography--h6 {
+      font-size: 0.75rem;
+    }
```

Comment:
this is no longer required

Reply:
Fixed in: f09cfff79ee9828090287ce51cb78ca11f56acdc